xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -116,18 +116,6 @@
       "minLength": 8,
       "defaultValue": "[concat(toUpper(uniqueString('cd-web')), '@', uniqueString(parameters('sqlserver_password')))]"
     },
-    "analytics_mongodb_connectionstring": {
-      "type": "securestring"
-    },
```

Comment:
I'm pretty sure mongo conn strings are used on deployment. You know, msdeploy.json is used for both vanilla and redeploy scripts. So I don't think those should be removed.